plugin/dnstap/dnstapio/io.go

```diff
@@ -1,13 +1,18 @@
 package dnstapio
 
 import (
-	"fmt"
 	"io"
+	"log"
+	"os"
 
 	tap "github.com/dnstap/golang-dnstap"
 	"github.com/golang/protobuf/proto"
 )
 
+func init() {
+	log.SetOutput(os.Stdout)
```

Comment:
I think this is already the default, it's also binary wide if you play with this setting.
Think it can be just removed.

Reply:
Oops, I wasn't it would affect the whole binary or just the package who imports.